plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1286,6 +1287,7 @@ blockly.block.wander=Wander around (enables walking) with speed factor %1 %2
 blockly.block.wander_avoid_water=Wander around and avoid water with speed factor %1 %2
 blockly.block.biome_entered=If player enters biome %1
 blockly.block.block_placed=If %1 is placed
+blockly.block.entity_hurt_player=If player gets hit by %3 with damage between %1 and %2 shield active: %4
```

Comment:
One more minor comment. Player can get hurt by other reasons than hitting, so it would be better to call this "If player gets hurt" than hit

Reply:
Updated